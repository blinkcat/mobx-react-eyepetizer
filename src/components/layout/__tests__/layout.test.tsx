import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Layout from '../';

describe('Component Layout', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Layout>test</Layout>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders title correctly', () => {
    const tree = renderer.create(<Layout title="Home">test</Layout>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
