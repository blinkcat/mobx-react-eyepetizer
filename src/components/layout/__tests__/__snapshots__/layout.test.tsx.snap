// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Layout renders correctly 1`] = `
<div
  className="app"
>
  test
  <style
    global={true}
    jsx={true}
  >
    
      * {
        box-sizing: border-box;
        outline: none;
      }
      body,
      html {
        height: 100%;
      }
      body {
        margin: 0;
        font-family: Lantinghei SC, Microsoft Yahei, Hiragino Sans GB, Helvetica Neue, Helvetica, Arial, sans-serif;
        font-size: 13px;
        line-height: 1.5;
      }
    
  </style>
</div>
`;

exports[`Component Layout renders title correctly 1`] = `
<div
  className="app"
>
  test
  <style
    global={true}
    jsx={true}
  >
    
      * {
        box-sizing: border-box;
        outline: none;
      }
      body,
      html {
        height: 100%;
      }
      body {
        margin: 0;
        font-family: Lantinghei SC, Microsoft Yahei, Hiragino Sans GB, Helvetica Neue, Helvetica, Arial, sans-serif;
        font-size: 13px;
        line-height: 1.5;
      }
    
  </style>
</div>
`;
